pkgs/applications/misc/gnuradio-ais/default.nix

```diff
@@ -12,8 +12,9 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "bistromath";
     repo = "gr-ais";
-    rev = "1863d1bf8a7709a8dfedb3ddb8e2b99112e7c872";
-    sha256 = "1vl3kk8xr2mh5lf31zdld7yzmwywqffffah8iblxdzblgsdwxfl6";
+    # Upstream PR: https://github.com/bistromath/gr-ais/commit/8502d0252a2a1a9b8d1a71795eaeb5d820684054
+    "rev" = "8502d0252a2a1a9b8d1a71795eaeb5d820684054";
```

Comment:
[1863d1b](https://github.com/bistromath/gr-ais/commit/1863d1bf8a7709a8dfedb3ddb8e2b99112e7c872) (Dec 20, 2015) is the newest commit on the repo, which is "unversioned"
[8502d02](https://github.com/bistromath/gr-ais/pull/18/commits/8502d0252a2a1a9b8d1a71795eaeb5d820684054) is just my pr that fixes a linking issue on top of the above. 
Although the version didn't change, should I change the version to the correct date? ("2015-12-20", the last commit to master)